core/templates/dev/head/editor/state_editor_content.html

```diff
@@ -13,6 +13,8 @@
       </span>
       <span angular-html-bind="content[0].value" class="protractor-test-state-content-display"></span>
     </div>
+    <!-- This is a dummy divison created to mask the contents when hovoverd over above division -->
```

Comment:
s/divison/division
s/hovoverd over above decision/the user hovers over the contents